bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -255,20 +262,23 @@ where
 		match dest_para_id {
 			Some(dest_para_id) => {
 				let dest_para_fee_asset: Asset = (Location::parent(), dest_para_fee).into();
+				let bridge_location = Location::new(2, GlobalConsensus(network));
 
 				instructions.extend(vec![
 					// Perform a deposit reserve to send to destination chain.
 					DepositReserveAsset {
-						assets: Definite(vec![dest_para_fee_asset.clone(), asset.clone()].into()),
+						assets: Definite(vec![dest_para_fee_asset.clone(), asset].into()),
 						dest: Location::new(1, [Parachain(dest_para_id)]),
 						xcm: vec![
 							// Buy execution on target.
 							BuyExecution { fees: dest_para_fee_asset, weight_limit: Unlimited },
-							// Deposit asset to beneficiary.
-							DepositAsset { assets: Definite(asset.into()), beneficiary },
+							// Deposit assets to beneficiary.
+							DepositAsset { assets: Wild(AllCounted(2)), beneficiary },
```

Comment:
To final destination beneficiary deposit unspent fees as well as the transferred asset.

Since the chain supports paying fees using `fee_asset`, it most surely also supports deposits for said asset.

Reply:
So Mythos accepts DOT for fees, but doesn't accept DOT deposits?

Reply:
Yeah, its weird. They have a custom asset trader that swaps dot in the message for MYTH to pay for execution. The collected DOT is basically discarded.

That said I think its their responsibility to configure their XCM AssetTransactor to just ignore deposited DOT. So I am happy with your approach.

I will tell them they have a 3 months to fix as necessary.